gap/PolygonalComplexes/constructors.gi

```diff
@@ -460,7 +460,7 @@ __SIMPLICIAL_IntSetConstructor("VerticesInFaces", __SIMPLICIAL_AllTypes,
 
 #######################################
 ##
-## UmbreallaDescriptors 
+##  UmbreallaDescriptors 
```

Comment:
Here is a typo.

Reply:
Thank you very much. t1*t2 moves around an edge and thus could contain cycles of length 2 or 4. This is checked in line 1736.